packages/kicadsch/kicadsch.0.2.0/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "Jean-Noel Avila <jn.avila@free.fr>"
+author: "Jean-Noel Avila <jn.avila@free.fr>"
+homepage: "https://jnavila.github.io/plotkicadsch/"
+bug-reports: "https://github.com/jnavila/plotkicadsch/issues"
+license: "ISC"
+dev-repo: "https://github.com/jnavila/plotkicadsch.git"
+build: [
+  [ "jbuilder" "subst" "-n" "kicadsch" ] {pinned}
```

Comment:
You may use the `name` variable rather than the hard coded package name here as well.

Reply:
Thanks for the comment. BTW the compilation failure is strange. Patience_diff is required in version v0.10.0 (in the opam file) but the CI keeps on using v0.9.0. Moreover, v0.10.0 is available in opam but is not present in this repo. 

I dont understand how all this works so I have no clue on how to solve this issue.